galeracluster/source/documentation/quorum-reset.rst

```diff
@@ -176,17 +176,17 @@ For servers that use ``systemd``, execute instead this from the command-line:
 
    systemctl stop mysql
 
-You're now ready to start the cluster again.  Start the most advanced node with the ``--wsrep-new-cluster`` option |---| not the other nodes.  For servers that use ``init``, run the following command:
+You're now ready to start the cluster again.  Start the most advanced node with the ``mysqld_bootstrap`` command |---| not the other nodes.  For servers that use ``init``, run the following command:
 
 .. code-block:: console
 
-   service mysql start --wsrep-new-cluster
+   service mysql start mysqld_bootstrap
 
 For servers that use ``systemd`` and Galera Cluster 5.5 or 5.6, enter instead the following from the command-line:

```

Comment:
Considering we haven't supported 5.5 and 5.6 for quite sometime, I suggest we start focusing on 5.7 and 8.0

Reply:
Changed